internal/controller/operator/factory/k8stools/client_utils.go

```diff
@@ -152,10 +152,7 @@ type ObjectWatcherForNamespaces struct {
 
 // NewObjectWatcherForNamespaces returns a watcher for events at multiple namespaces  for given object
 // in case of empty namespaces, performs cluster wide watch
-func NewObjectWatcherForNamespaces[T any, PT interface {
```

Comment:
T and PT are well-known shorthand  for Type and PointerType, what is a reason of this change?

Reply:
it's done for consistency
https://github.com/VictoriaMetrics/operator/blob/25dbe3c8d0b7109e1b12be599d790734674bf202/internal/controller/operator/factory/k8stools/client_utils.go#L113
https://github.com/VictoriaMetrics/operator/blob/25dbe3c8d0b7109e1b12be599d790734674bf202/internal/controller/operator/factory/k8stools/selectors.go#L35

Reply:
updated in all places